src/auto/responses.rs

```diff
@@ -2,55 +2,66 @@ use std::error::Error;
 
 use serde::{Deserialize, Serialize};
 
-use crate::{Message, AgentInfo};
+use crate::{AgentInfo, Message};
 
 use super::try_parse_json;
 
 #[derive(Serialize, Deserialize)]
 pub struct Response {
-    response: String
+    response: String,
 }
 
 pub fn create_runner_prompt() -> String {
-    format!(
-r#"Now, please write a response back to the user. Tell the user, in detail, everything you did, the outcome, and any permanent changes that were carried out."#)
+    format!("Write a concise response detailing actions, outcomes, and any lasting changes.")
 }
 
 pub fn ask_for_responses(agent: &mut AgentInfo) -> Result<String, Box<dyn Error>> {
-    agent.llm.message_history.push(Message::User(create_runner_prompt()));
+    agent
+        .llm
+        .message_history
+        .push(Message::User(create_runner_prompt()));
 
-    let response = agent.llm.model.get_response_sync(
-        &agent.llm.get_messages(), Some(1000), None
-    )?;
+    let response =
+        agent
+            .llm
+            .model
+            .get_response_sync(&agent.llm.get_messages(), Some(1000), None)?;
 
     Ok(response)
 }
 
 pub fn create_assistant_prompt(context: &str, request: &str) -> String {
     format!(
-r#"Given the findings from your task, and this conversation context:
+        r#"
+Considering the task findings and conversation context:
 
 {context}
 
-Write a response back to the user. The original message they sent was: "{request}"
-Reply in this JSON format: 
+Respond concisely to: "{request}", using the JSON format below, and include the source of your information in parentheses:
 
 {{
-    "response": "..."
+    "response": "<Your Response (Source)>"
 }}
-
-Respond in that exact JSON format exactly.
-
-Provide a response as an assistant to the initial request in the above format.
-Make sure you include where you got the information from in your response, in parantheses."#)
+"#
+    )
```

Comment:
limited test shows that this shortened version is enough to make llm produce consistent output as original version